### DISCUSSION TITLE
Realizar video Demo

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#514] [Ideas]
De que va a hablar cada uno y que hay que tener encuenta sobre el tema elegido

**Comment by NataliaAlvarezIspc** [comment]
Puedo hablar de Registración y Login

**Reply by rpgrca** [reply]
Ideas a cubrir

- Hay que hablar que el superuser creado es el administrador, que por el momento no se pueden registrar ni crear más administradores
- Que todas las cuentas que se registran son clientes
- Validación de datos
- Login incorrecto, mostrar mensaje de error
- Login correcto como administrador, mencionar el header
- Mencionar cookie por un día hecho en JWT
- Logout
- Login correcto como cliente que recién creaste, mostrar header
- Logout
- Mencionar lo de la recuperación de clave
- Mostrar el entry point de django de Login, Logout y Signup

**Reply by rpgrca** [reply]
También ya que estás como administrador podés mostrar la lista de Usuarios registrados.

**Comment by BraianTroncoso** [comment]
Puedo hablar del Catalogo y Home

**Reply by rpgrca** [reply]
Ideas a cubrir
- Logueado como cliente, mostrar el Home, la api externa que se consume del sprint pasado, no mencionar mucho lo de que hay de nuevo porque está todo hardcodeado
- Catálogo, listado de productos existentes, mostrar el mensaje GET
- Entrar como administrador y crear un producto para que aparezca en el catálogo un producto más, mostrar el mensaje POST
- Modificar nombre de un producto y que se vea reflejado en el home, mostrar el mensaje PUT
- Mencionar la intención del botón agregar para el próximo sprint
- Borrar con el administrador todos los productos para mostrar el mensaje que no hay ningún inventario, mostrar el mensaje DELETE
- Mostrar el entry point de artículos de django

**Comment by Brian-Almada** [comment]
Puedo hablar de Contacto, Perfil y Compras de un usuario común.
(estaba por elegir carrito también pero creo que no lo entiendo tan bien todavía)

**Reply by rpgrca** [reply]
Ideas a cubrir
- Contacto enviará un mail al administrador, llena nombre y email con información de usuario si está logueado, si no hay que llenar datos a mano
- Carrito de compras, que ahí se vera lo que se fue agregando desde el catálogo.
- Opciones de envío
- Opciones de pago y que vamos a usar la API de Mercado Pago para ese pago.
